zerver/tests/test_bots.py

```diff
@@ -11,7 +11,7 @@
 
 from zerver.lib.actions import do_change_stream_invite_only
 from zerver.models import get_realm, get_stream, \
-    Realm, Stream, UserProfile, get_user, get_bot_services, Service
+    Realm, Stream, UserProfile, Recipient, get_user, get_bot_services, Service
```

Comment:
This change is probably in the wrong commit or should be removed (at least, this commit doesn't use `Recipient`)